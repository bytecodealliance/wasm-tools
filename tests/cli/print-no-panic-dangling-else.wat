;; RUN: print %

(module
    (func else)
)
