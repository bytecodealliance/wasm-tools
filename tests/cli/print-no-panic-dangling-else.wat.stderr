error: `else` found outside `If` block (at offset 0x18)
